include/solvers/heat_transfer_scratch_data.h

```diff
@@ -99,13 +98,15 @@ class HeatTransferScratchData
                           const Quadrature<dim>          &quadrature,
                           const Mapping<dim>             &mapping,
                           const FiniteElement<dim>       &fe_fd,
-                          const Quadrature<dim - 1>      &face_quadrature)
+                          const Quadrature<dim - 1>      &face_quadrature,
+                          const double                   &delta_T_ref)
```

Comment:
not sure it's worth it to pass a reference to a double, just pass by copy instead.